main.js

```diff
@@ -0,0 +1,39 @@
+
+(function() {
+    const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
+    const themeToggleButton = document.querySelector('.theme-toggle');
+    const darkSchemePreference = prefersDarkScheme ? 'dark' : 'light';
+    const currentTheme = localStorage.getItem('theme') ? localStorage.getItem('theme') : darkSchemePreference;
+    const rootElement = document.querySelector(':root');
```

Comment:
It looks like `:root` resolves to the `html` element instead of the `body`. I wonder if we should just use `body` everywhere we're using root? The class doesn't seem to be applying correct onto the `html` element.

Reply:
Wait never mind I think it's still just the issue where it doesn't work if you start off from light.